src/BloomBrowserUI/bookLayout/basePage.less

```diff
@@ -257,17 +257,27 @@ preview.css.
 //currently, in edit mode every page looks like page 1 to css, so left-vs-right margins are just confusing
 //so we only turn them on when making PDF
 .publishMode :not(.outsideFrontCover):not(.outsideBackCover){
-  &.bloom-page:nth-of-type(odd) .marginBox  {
-	  left: @MarginOuter;
-  }
-  &.bloom-page:nth-of-type(even) .marginBox  {
-	  left: @MarginInner;
-  }
+	&.bloom-page:nth-of-type(odd){
```

Comment:
The A6 booklets seem to be folded "horizontally" (i.e. not calendar-folded) for both Portrait and Landscape, so they fall under the :not(.A5Landscape):not(.A4Landscape) sections and get Outer or Inner margins like the other non-Landscape layouts.